web/src/components/App.tsx

```diff
@@ -65,18 +58,14 @@ class App extends React.Component<Props, State> {
     }
 
     if (RemoteData.isSuccess(errors)) {
-      let decodedErrorKey = decodeURIComponent(match.params.errorKey)
-      let activeError = errors.data.find(e => e.aggregation_key === decodedErrorKey)
-      if (
-        activeError !== undefined &&
-        (activeError !== decodedErrorKey) &&
```

Comment:
I noticed that here might have been a bug, because `activeError` and `decodedErrorKey` are different data types (AggregatedError and string) respectively, so this part of the condition was always true.

So I've changed it and also the condition handling, to what I think, is a bit more declarative.

Reply:
Nice catch! I think the new condition is much better but there might an issue with the `return` statement there.

Reply:
Could you elaborate on the possible issue ?

Reply:
Oh sorry, I think I was only confused by the syntax. All good 👍